docs/testing-guidelines/WritingPesterTests.md

```diff
@@ -100,6 +100,11 @@ In the AppVeyor CI, we run two different passes:
 
 In each case, tests are executed with appropriate privileges.
 
+Tests that need to be run with sudo **on unix systems** should be additionally marked with 'RequireSudoOnUnix' Pester tag.
+Similarly as above, we run the tests in Travis CI in two passes:
+- with sudo and 'RequireSudoOnUnix' tag only
```

Comment:
I think this should be worded as: With sudo and only tests with `RequireSudoOnUnix` tag